internal/producer/src/validation/validation.go

```diff
@@ -48,8 +50,15 @@ func ValidateContract(c *contracts.Contract) error {
 	}
 
 	// retrieve sender's balance from account balance table
+	dbConnection, err := sql.Open("sqlite3", constants.AccountsTable)
```

Comment:
Can we pass this in as a parameter instead?

Reply:
sure